src/data.json

```diff
@@ -0,0 +1,4 @@
+{
+    "username": "admin",
+    "post": "sic transit gloria mundi"
+}
```

Comment:
is this related?

Reply:
no, this is from for another, secret pr 👀